IdokladSdk/Requests/Core/Modifiers/Filters/Common/ContainTagIdFilterItem.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections.Generic;
+
+namespace IdokladSdk.Requests.Core.Modifiers.Filters.Common
+{
+    /// <summary>
+    /// Contain TagId filter item.
+    /// </summary>
+    public class ContainTagIdFilterItem : FilterItemBase
```

Comment:
Proč nemůže ContainTagIdFilterItem dědit z ContainArrayFilterItem? Měl by navíc akorát metody IsEqual a IsNotEqual.

Reply:
filter pre TagId môže použiť len operátor contains, iné operátory v  API pre TagId nepodporujeme